src/check_email.sh

```diff
@@ -64,12 +64,8 @@ is_quicinc_allowed() { # role
         Author) property="allow-quicinc-authors" ;;
         *) return 2 ;; # Not a valid role, programming error
     esac
-    echo "$CUSTOM_PROPERTIES_JSON" | jq -e '
-      any(.[];
-        .property_name == "'"$property"'" and
-        .value == "true"
-      )
-    ' > /dev/null
+    echo "$CUSTOM_PROPERTIES_JSON" | jq -e --arg property "$property" \
+    'any(.[]; .property_name == $property and .value == "true")' > /dev/null
```

Comment:
nit, missing an indent here
```suggestion
        'any(.[]; .property_name == $property and .value == "true")' > /dev/null
```

Reply:
Done